packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -985,6 +985,10 @@ export const bookingsRouter = router({
 
       await sendDeclinedEmails(evt);
     }
-    return { message: "Booking " + confirmed ? "confirmed" : "rejected" };
+
+    const message = "Booking " + confirmed ? "confirmed" : "rejected";
+    const status = confirmed ? BookingStatus.ACCEPTED : BookingStatus.REJECTED;
+
+    return { message, status };
```

Comment:
earlier { message: "Booking " + confirmed ? "confirmed" : "rejected" }
returned only {message : "confirmed" } everytime for some reason. doing concatenation before fixed it